### ISSUE TITLE
Unable to deploy

### ISSUE THREAD

**Issue by david--wright** [defenseunicorns/zarf-package-software-factory#32]
Attempting to deploy this package with the default settings results warnings during deploy and image pull backoff errors for all images.

### Environment: 
ec2 c5a.8xlarge instance running RHEL 7.9

### Commands Ran:
```bash
yum update -y
yum install -y git wget 
sysctl -w vm.max_map_count=262144
mkdir -p /work/zarf/bin
git clone https://github.com/defenseunicorns/zarf-package-software-factory.git /work/factory
wget https://zarf-public.s3-us-gov-west-1.amazonaws.com/release/v0.14.0/zarf -O /work/zarf/bin/zarf
wget https://zarf-public.s3-us-gov-west-1.amazonaws.com/release/v0.14.0/zarf-init.tar.zst -O /work/zarf/bin/zarf-init.tar.zst
chmod +x /work/zarf/bin/zarf
export PATH="$PATH:/work/zarf/bin:/usr/local/bin"
echo 'export PATH=$PATH:/work/zarf/bin:/usr/local/bin' >> ~/.bash_profile
cd /work/zarf/bin
curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
zarf init --confirm --components management,gitops-service --host 127.0.0.1 
mv /.git-credentials /root/.git-credentials
zarf tools registry login registry1.dso.mil -u <user_name> -p <some-password>
cd /work/factory
make build
cd build
zarf package deploy zarf-package-software-factory.tar.zst --confirm
```

### Warnings:
INFO[0008] Updating image                                source="registry1.dso.mil/ironbank/fluxcd/helm-controller:v0.11.0" target=
WARN[0010] Unable to push the image to the registry      source="registry1.dso.mil/ironbank/fluxcd/helm-controller:v0.11.0" target=
INFO[0010] Updating image                                source="registry1.dso.mil/ironbank/fluxcd/kustomize-controller:v0.13.0" target=
WARN[0011] Unable to push the image to the registry      source="registry1.dso.mil/ironbank/fluxcd/kustomize-controller:v0.13.0" target=
INFO[0011] Updating image                                source="registry1.dso.mil/ironbank/fluxcd/notification-controller:v0.15.0" target=
WARN[0012] Unable to push the image to the registry      source="registry1.dso.mil/ironbank/fluxcd/notification-controller:v0.15.0" target=
INFO[0012] Updating image                                source="registry1.dso.mil/ironbank/fluxcd/source-controller:v0.14.0" target=
WARN[0013] Unable to push the image to the registry      source="registry1.dso.mil/ironbank/fluxcd/source-controller:v0.14.0" target=

### Errors:
 Warning  Failed     9m8s (x4 over 10m)   kubelet            Failed to pull image "registry1.dso.mil/ironbank/fluxcd/helm-controller:v0.11.0": rpc error: code = Unknown desc = failed to pull and unpack image "registry1.dso.mil/ironbank/fluxcd/helm-controller:v0.11.0": failed to resolve reference "registry1.dso.mil/ironbank/fluxcd/helm-controller:v0.11.0": pulling from host 127.0.0.1 failed with status code [manifests v0.11.0]: 401 Unauthorized

Deploying other packages/examples does not result in the warning and lists the target as 127.0.0.1 rather than blank as seen here.

**Comment by RothAndrew** [comment]
The `target=<blank>` in the logs leads me to believe this is the same issue as https://github.com/defenseunicorns/zarf/issues/347. 

> Sometimes, usually after a zarf destroy, the .zarf-state.yaml and $HOME/.git-credentials files will get into a bad state, where the IP address / hostname is not present in the configuration, and zarf package deploy fails because it doesn't know where to push the docker images. This issue is resolved in v0.15.0 and can be mitigated on v0.14.0 by deleting .zarf-state.yaml and $HOME/.git-credentials (assuming there isn't other non-zarf stuff in it) after running zarf destroy.

I'm very close to having #30 done. Once it is, the package will be compatible with Zarf v0.15.0 which is not affected by this issue.

Something else you can try is moving `zarf-package-software-factory.tar.zst` to the `/work/zarf/bin` directory where you ran `zarf init`. Zarf v0.14.0 requires `zarf init` and `zarf package deploy` to have been run from the same directory because of the way it does the `.zarf-state.yaml` file. This is also resolved in v0.15.0.

**Comment by RothAndrew** [comment]
@david--wright per our last conversation I believe this is resolved after moving to Zarf v0.15.0. Closing. Please let me know if there's any outstanding issues so we can reopen.
